apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/AplCore.java

```diff
@@ -220,18 +229,26 @@ private void startUp() {
                 ShufflingParticipant.init();
                 PrunableMessage.init();
                 TaggedData.init();
-                AppStatus.getInstance().update("Peer server initialization...");
+                aplAppStatus.durableTaskUpdate(initCoreTaskID,  60.0, "Apollo Account ledger initialization done");
                 Peers.init();
-                AppStatus.getInstance().update("API Proxy initialization...");
                 APIProxy.init();
                 Generator.init();
                 AddOns.init();
-                AppStatus.getInstance().update("API initialization...");
                 Helper2FA.init(databaseManager);
+                aplAppStatus.durableTaskUpdate(initCoreTaskID,  70.1, "Apollo cor cleaases initialization done");
 //signal to API that core is reaqdy to serve requests. Should be removed as soon as all API will be on RestEasy                
                 ApiSplitFilter.isCoreReady = true;
 
-                ThreadPool.scheduleThread("Active connections logger", () -> LOG.debug("Used connections - '{}'", databaseManager.getDataSource().getJmxBean().getActiveConnections()), 15, TimeUnit.SECONDS);
+                ThreadPool.scheduleThread("DB_con_log_AplAppStatus_clean", 
+                   new Runnable() {
```

Comment:
Better to use lamda expression 
`ThreadPool.scheduleThread("Active connections logger", () -> {
LOG.debug("Used connections - '{}'", databaseManager.getDataSource().getJmxBean().getActiveConnections());
aplAppStatus.clearFinished(10*60L); //10 min
}
, 15, TimeUnit.SECONDS)`

Reply:
It is matter of taste, my is explicit Runnable